plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -333,6 +333,100 @@ public void testPredicatePushdown(String value, String operator)
         }
     }
 
+    @Test(dataProvider = "predicatePushdownProvider")
```

Comment:
Additionally can we enable this flag true for all our query runners and check if it passes the CI